firstvoices/backend/views/custom_order_recalculate_views.py

```diff
@@ -26,6 +32,11 @@
             403: OpenApiResponse(description="Todo: Not authorized for this Site"),
             404: OpenApiResponse(description="Todo: Site not found"),
         },
+        parameters=[
+            OpenApiParameter(
```

Comment:
should we add this OpenAPIParameter to a variable ? will help if we need to modify the param later since its being reused throughout the file